src/background.js

```diff
@@ -322,7 +322,29 @@ function stripFbclid(url) {
   return strippedUrl.href;
 }
 
+async function tabUpdateListener (tabId, changeInfo, tab) {
+  await updateBrowserActionIcon(tab.url);
+}
+
+async function updateBrowserActionIcon (url) {
+  if(isFacebookURL(url)) {
+    browser.browserAction.setPopup({popup: "./panel1.html"});
+    const FBC_STORAGE = await browser.storage.local.get();
+    if (FBC_STORAGE.PANEL_SHOWN !== true) {
+      await browser.browserAction.setBadgeBackgroundColor({
+        color: "#3B5998"
+      });
+      browser.browserAction.setBadgeText({text: "funny"});
+    }
+  } else {
+    browser.browserAction.setPopup({popup: "./panel2.html"});
+    browser.browserAction.setBadgeText({text: ""});
+  }
+}
+
 async function containFacebook (options) {
+  await updateBrowserActionIcon (options.url);
```

Comment:
The space after the function name declaration I find a weird but it's used elsewhere. However the space on call sites appears not to be.
The current style appears to be:
```
function name () {
}
name();
```